<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pip Installer Gui (PIG)</string>
  </property>
  <widget class="QWidget" name="main_window_container">
   <widget class="QWidget" name="internal_container" native="true">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>821</width>
      <height>611</height>
     </rect>
    </property>
    <widget class="QPushButton" name="btn_open_folder_files">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>150</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>choose wheel</string>
     </property>
    </widget>
    <widget class="QLabel" name="config_label">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>20</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Pip Installer Gui</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>TINTING IP</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>150</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>choose wheel to install</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="qline_folder_path">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>150</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="qline_ip">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="config_label_2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>100</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Base Settings</string>
     </property>
    </widget>
    <widget class="QLabel" name="config_label_3">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>200</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Advance Settings</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_install">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>370</y>
       <width>85</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>INSTALL</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_validate">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>370</y>
       <width>85</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>VALIDATE IP</string>
     </property>
    </widget>
    <widget class="QLabel" name="version_lbl">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>60</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_venvs">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>260</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>240</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Choose a virtualenv</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_clear">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>370</y>
       <width>85</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>CLEAR ALL</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>240</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Ignore Requires</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_requires">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>260</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_board">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>420</y>
       <width>761</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
